src/main/java/core/basesyntax/randomizer/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.randomizer;
+
+import core.basesyntax.model.Colors;
+import java.util.Random;
+
+public class ColorSupplier {
+    private int index = new Random().nextInt(Colors.values().length - 1);
```

Comment:
I would recommend to make `index` a local variable in this case, as it is not constant nor class dependency. Also, fell free to do 
`.nextInt(Color.values().length);`
Because `nextInt(int bound)` method generates values between 0 (inclusive) and the specified value (exclusive). 
[Java Doc](https://docs.oracle.com/javase/8/docs/api/java/util/Random.html#nextInt-int-)